spec/globus/client_spec.rb

```diff
@@ -83,35 +83,32 @@
     end
   end
 
-  describe "#mkdir" do
+  context "when invoking client methods" do
```

Comment:
I rearranged these tests to avoid duplication. Some of this may not be necessary or get cleaned up later, but it was necessary to maintain test coverage.